javaparser-core/src/main/java/com/github/javaparser/ast/ArrayCreationLevel.java

```diff
@@ -76,11 +76,13 @@ public ArrayCreationLevel(TokenRange tokenRange, Expression dimension, NodeList<
     }
 
     @Override
+    @Generated("com.github.javaparser.generator.core.node.AcceptGenerator")
```

Comment:
I would love a tool to identify these changes and hide them :D

Reply:
There must be about 400 of them? :-D